client/webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    mode: 'development',
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'bundle.js',
+        publicPath: '/public/'
+    },
+    resolve: {
+        modules: ['node_modules'],
+        extensions: ['.js', '.json', '.jsx', '.css', '*']
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader'
+                }
+            },
+            {
+                test: /\.html$/,
+                use: [
+                    {
+                        loader: 'html-loader'
+                    }
+                ]
+            },
+            {
+                test: /\.css$/,
+                use: [
+                    'style-loader',
+                    {
+                        loader: 'css-loader',
+                        options: {
+                            modules: true
+                        }
+                    }
+                ]
+            },
+            {
+                test: /\.(png|svg|jpg|gif|ico)$/,
+                use: ['file-loader']
+            }
+        ]
+    },
+    plugins: [
+        new HtmlWebPackPlugin({
+            template: './public/index.html',
+            filename: './index.html'
+        })
+    ],
+    devServer: {
+        contentBase: path.join(__dirname, 'dist'),
+        compress: true,
+        port: 3000,
+        watchContentBase: true,
+        progress: true,
+        proxy: {
+            '/api': {
```

Comment:
Not sure exactly (could be wrong) but I think the proxy in the [package.json](https://github.com/dimaka77/myDashboard/pull/2/files#diff-52dbbf94d5c54b68c0cad0702ba94a96R5) takes care  of the port proxying so that we should be able to execute API calls to our server by just calling the endpoint by name (e.g. `fetch('signup')`)